src/core/OmnichannelErrorCodes.ts

```diff
@@ -1,5 +1,6 @@
 enum OmnichannelErrorCodes {
     WidgetUseOutsideOperatingHour = 705,
+    noWidgetPresentErrorCode = 701,
```

Comment:
WidgetNotFound?

It's already implied that it's an error code based on the name `OmnichannelErrorCodes`. So, `error code` can be removed from the naming.